crowbar_framework/app/controllers/nodes_controller.rb

```diff
@@ -139,6 +139,11 @@ def bulk
               dirty = true
             end
 
+            unless node.default_fs == node_attributes["default_fs"]
```

Comment:
@jsuchome the issue is that if you set the default fs of a node, then go to the bulk edit page, then you'll trigger this code and you will lose the default fs that was set.